index.html

```diff
@@ -21,86 +21,243 @@
+                    required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <ul class="styling3">
+                    <li>css</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>html</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>bootstrap</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>Ruby</li>
+
+
+                </ul>
+
+                <button type="button" class="button1">See project</button>
+
+
+            </div>
+
+
+            <div class="redifining">
+                <img class="picture" src="Img.Placeholder.svg" alt="picture">
+                <h2 class="text">Multi-Post Stories</h2>
+                <p class="paragraph-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <ul class="styling3">
+                    <li>css</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>html</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>bootstrap</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>Ruby</li>
 
+
+                </ul>
+
+                <button type="button" class="button1">See project</button>
+
+
+            </div>
+
+
+            <div class="redifining">
+                <img class="picture" src="Img.Placeholder.svg" alt="picture">
+                <h2 class="text">Multi-Post Stories</h2>
+                <p class="paragraph-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <ul class="styling3">
+                    <li>css</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>html</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>bootstrap</li>
+                    <li><img src="Rectangle.70.svg" alt="border"></li>
+                    <li>Ruby</li>
+
+
+                </ul>
+
+                <button type="button" class="button1">See project</button>
+
+
+            </div>
+        </section>
+
+        <section class="image">
+            <h1 class="firsttext">CONTACT US</h1>
```

Comment:
- [ ] The contact section here is not aligned similar to that of the Figma template. Please follow all the alignment as seen in the Figma
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/170707402-ebf47114-513d-4ccf-8918-a5bcdc49e68a.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/170707478-49591b08-e2c0-407c-a3cf-9b7a2300e88a.png)

Reply:
Hi Sir, 

This part of the project is not required yet, I added it by accident.
So I need to remove it.
Hope you understand, thank you!

Best regards.

-Jesse McCallums